debian/rules

```diff
@@ -132,5 +132,8 @@ override_dh_systemd_start:
 	# stopping the service during upgrades.
 	dh_systemd_start --package=docker.io -r
 
+override_dh_installsystemd:
+	dh_installsystemd --name=docker
```

Comment:
We probably want to move this up next to `override_dh_installinit` so it's more obvious we're missing `--no-start` on it, right? :innocent: :grimacing:

Reply:
(Otherwise this all LGTM! :metal:)

Reply:
Thanks for catching this! :-)

I moved it up, and now it is below `override_dh_installinit`. Also added the missing `--no-start` option.